RunGcm-filters-leticia.ipynb

```diff
@@ -0,0 +1,3088 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #plotting
</pre><p>could modify this comment to:</p><pre class="ql-syntax" spellcheck="false"># filter computation for one time slice and plotting
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oceanhackweek/ohw22-proj-gcmfilters/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>